initdb.py

```diff
@@ -858,6 +862,14 @@ def populate_database(minimal=False):
     org.stripe_id = TEST_STRIPE_ID
     org.save()
 
+    QuotaType.create(name="Notify")
```

Comment:
These names don't match those created in the alembic migration. Is this intended?